config_builder/materializer.py

```diff
@@ -45,8 +49,21 @@ def materialize_file(
     materialized_path = root_dir / materialized_root / relative_path
     os.makedirs(materialized_path.parent, exist_ok=True)
 
+    if ext_package != "":
+        imported_module = importlib.import_module(ext_package)
+        if not hasattr(imported_module, "__file__") or imported_module.__file__ is None:
+            raise ValueError(f"Cannot determine path for module {ext_package}")
+        ext_package_path = Path(imported_module.__file__).parent.resolve()
+        import_paths = [ext_package_path]
+    else:
+        import_paths = []
+
```

Comment:
See my comment on the jsonnet-lib pr. Let's either add a custom loader or refactor the api to make import absolute.